CMakeLists.txt

```diff
@@ -0,0 +1,121 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is it cherry-picked from `hunter-0.10.0` branch? Can you add a note about it? Is there any difference?

Reply:
CMake build seems to be new. Essentially checked out all CMake stuff from 0.10.0, then changed source files to match 0.9.2. Where should I add the note?

Reply:
> Essentially checked out all CMake stuff from 0.10.0

Commit 1: "CMake stuff from hunter-0.10.0 branch"

> then changed source files to match 0.9.2

Commit 2 (I guess it's https://github.com/hunter-packages/thrift/pull/3/commits/1d24cc93a10a721a27de8f88fcb2933ff9818ad9 already?)